documentation/tut/toy-example-reporter.rst

```diff
@@ -0,0 +1,241 @@
+.. _tut_toy_schedule_reporter:
+
+Toy example IV: Computing reports
+=====================================
+
+So far, we have worked on the scheduling batteries and processes. Now, we are moving to one of the other three pillars of FlexMeasures: reporting. 
+
+In essence, reporters apply arbitrary transformations to data coming from some sensors (multiple inputs) and save the results to other sensors (multiple outputs). In practice, this allows to compute KPIs (e.g. profit, total daily energy production, ...), apply operations to beliefs (e.g. change sign of a power sensor for a time period), among other things. 
+
+Currently, FlexMeasures comes with the following reporters:
+    - `PandasReporter`: applies arbitrary pandas methods to sensor data. 
+    - `AggregatorReporter`: combines data from multiple sensors into one using any of the Pandas `aggregate` function supported method (e.g. sum, average, max, min...).
+    - `ProfitLossReporter`: computes the profit/loss due to an energy flow under a specific tariff.
+
+Moreover, it's possible to implement your custom reporters in plugins. Instructions for this to come.
+
+Now, coming back to the tutorial, we are going to use the `AggregatorReporter` and the `ProfitLossReporter`. In the first part, we'll use the `AggregatorReporter` to compute the (discharge) headroom of the battery in :ref:`tut_toy_schedule_expanded`. That way, we can verify the maximum power at which the battery can discharge at any point of time. In the second part, we'll use the `ProfitLossReporter` to compute the profit of operating the process of Tut. Part III in the different policies.
+
+Before getting to the meat of the tutorial, we need to set up up all the entities. Instead of having to do that manually (e.g. using commands such as ``flexmeasures add sensor``), we have prepared a command that does that automatically.
+
+Setup
+.....
+
+Just as in previous sections, we need to run the command ```flexmeasures add toy-account``, but this time with a different value for *kind*:
+
+.. code-block:: bash
+
+    $ flexmeasures add toy-account --kind reporter
+
+Under the hood, this command is adding the following entities:
+    - A yearly sensor that stores the capacity of the grid connection.
+    - A power sensor, `headroom`, to store the remaining capacity for the battery. This is where we'll store the report.
+    - A `ProfitLossReporter` configured to use the prices that we setup in Tut. Part II.
+    - Three sensors to register the profit of running the process of Tut. Part III.
```

Comment:
Three sensors to register the profits/losses from running the three different processes of Tut. Part III.